project/Dependencies.scala

```diff
@@ -9,9 +9,10 @@ object Dependencies {
   val scalaTestV    = "3.0.1"
   val slickV        = "3.2.1"
 
-  val workbenchUtilV   = "0.2-d34dcf2"
-  val workbenchModelV  = "0.1-d34dcf2"
-  val workbenchGoogleV = "0.1-d34dcf2"
+  val workbenchUtilV    = "0.2-544dd49-SNAP"
```

Comment:
Assuming you will update these to a non `-SNAP` after merging libs

Reply:
yes